src/resources/api/config.js

```diff
@@ -3,44 +3,45 @@ import beforeRequest from "./utils/beforeRequest";
 import handleError from "./utils/handleError";
 
 function createResource() {
-  const instance = axios.create({
-    headers: {
-      Accept: "application/json",
-      "Content-Type": "application/json",
-    },
-  });
-
-  instance.interceptors.request.use(
-    (config) => {
-      const token = localStorage.getItem("token");
-      if (token) config.headers.Authorization = "Bearer " + token;
-      beforeRequest(config);
-
-      return config;
-    },
-    (error) => {
-      return Promise.reject(error);
-    }
-  );
-
-  instance.interceptors.response.use(
-    (response) => {
-      return Promise.resolve(response.data);
-    },
-    (error) => {
-      return handleError(error);
-    }
-  );
-
-  return instance;
+    const instance = axios.create({
+        headers: {
+            Accept: "application/json",
+            "Content-Type": "application/json",
+        },
+    });
+
+    instance.interceptors.request.use(
+        (config) => {
+            //const token = localStorage.getItem("token");
+            const token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwOlwvXC9sb2NhbGhvc3RcL2FkbWluXC92MVwvYXV0aFwvbG9naW4iLCJpYXQiOjE2NDcxMDMzNDIsImV4cCI6MTY0OTY5NTM0MiwibmJmIjoxNjQ3MTAzMzQyLCJqdGkiOiJxNXRKeENxMTd0REE1aGlEIiwic3ViIjoxLCJwcnYiOiI3NjFkYmE4MDAwOThmNDYzNmVkMGE4NWY5NjNlOTk3MjY1MjhjZmYxIn0.7twAsaH2-INKqCv30wBbcC-tmIeRVO-Vky5K0F1XNFw"
```

Comment:
i think you mistakenly pushed a change you made for local use, don't forget to change this back ^^